### ISSUE TITLE
Compiled binary is stuck on Windows

### ISSUE THREAD

**Issue by ZetaoYang** [flameshot-org/flameshot#1362]
**Flameshot version**
Since "fixing icon regression (#1051)"   f11073bc00f1f656a590daa940e7497400f273cb

**Describe the bug**
When I compile in Jetbrains clion on my local Windows machine, the compiled binary is stuck and cannot be opened. 

**The root cause**
There is no exact equivalent for POSIX symlinks on Windows. These files are symlinks,
```
 data/img/app/flameshot.png
 data/img/app/flameshot.svg
 data/img/hicolor/128x128/apps/flameshot.png
 data/img/hicolor/48x48/apps/flameshot.png
 data/img/hicolor/scalable/apps/flameshot.svg
```
Because the default system [core.symlinks](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresymlinks) setting for [git-for-windows](https://github.com/git-for-windows/git) is `false`.
```pwsh
❯ git config --system --list
core.symlinks=false
core.autocrlf=true
core.fscache=true
color.diff=auto
color.status=auto
color.branch=auto
color.interactive=true
help.format=html
diff.astextplain.textconv=astextplain
rebase.autosquash=true
filter.lfs.clean=git-lfs clean -- %f
filter.lfs.smudge=git-lfs smudge -- %f
filter.lfs.process=git-lfs filter-process
filter.lfs.required=true
credential.helper=helper-selector
```
More details, see https://github.com/git-for-windows/git/wiki/Symbolic-Links.

However, when I go to run `git clone -c core.symlinks=true git@github.com:flameshot-org/flameshot.git` as a non-admin user ,
```pwsh
❯ git clone -c core.symlinks=true git@github.com:flameshot-org/flameshot.git
Cloning into 'flameshot'...
remote: Enumerating objects: 46, done.
remote: Counting objects: 100% (46/46), done.
remote: Compressing objects: 100% (43/43), done.
remote: Total 11478 (delta 29), reused 8 (delta 3), pack-reused 11432
Receiving objects: 100% (11478/11478), 11.54 MiB | 3.40 MiB/s, done.
Resolving deltas: 100% (8349/8349), done.
error: unable to create symlink data/img/app/flameshot.png: Permission denied
error: unable to create symlink data/img/app/flameshot.svg: Permission denied
error: unable to create symlink data/img/hicolor/128x128/apps/flameshot.png: Permission denied
error: unable to create symlink data/img/hicolor/48x48/apps/flameshot.png: Permission denied
error: unable to create symlink data/img/hicolor/scalable/apps/flameshot.svg: Permission denied
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```
Must run as administrator,
![image](https://user-images.githubusercontent.com/10769951/108582835-977cc500-7370-11eb-92b6-38b85ceff691.png)

**Q**: Some people may ask me why the `flameshot.exe` compiled in github actions can run normally?
**A**: Because This https://github.com/actions/virtual-environments/pull/1186 with `core.symlinks=true` by default, check this comment https://github.com/actions/virtual-environments/issues/1177#issuecomment-654181951.
For appveyor ci, see https://github.com/appveyor/ci/issues/650.

**Expected behavior**
After compiling, `flameshot.exe` can be opened normally. 

**System Information**
Windows 10 20H2, MSVC build environment

To sum up, I think flameshot as a cross-platform software, the main developer should consider whether your solution is available on other platforms when dealing with some problems.

**Comment by ZetaoYang** [comment]
I think there are two solutions, 
1. write down this situation in the documentation. For developers on Windows platform, it is especially important to run `git clone -c core.symlinks=true git@github.com:flameshot-org/flameshot.git` as an administrator to initialize the project. 
2. replace those symbolic links with normal image files of the same name. This may cause all jobs in github actions to fail with the error "git Error: unable to create symlink, File name too long", set `git config core.symlinks false` in workflow yaml to fix it.

**Comment by mmahmoudian** [comment]
I haven't developed anything for over a decade on Windows, therefore I might not have a say in this. But imho the second solution is more fool-proof and doesn't need extra documentation. These files are few and small in size. The only drawback I can think of is changing the icon would need change in two places, which I don't think it is a big deal.

**Comment by borgmanJeremy** [comment]
Ugh windows makes me sad:(  I can't believe you need either Admin or Dev mode for symlinks

I am fine duplicating the icons to make the build process easier. As @mmahmoudian these files are small and change so rarely having copies likely will not be an issue.
